src/containers/ConceptPage/components/ConceptForm.jsx

```diff
@@ -145,15 +145,27 @@ class ConceptForm extends Component {
       creators: values.creators,
       processors: values.processors,
       rightsholders: values.rightsholders,
+      copyright: {
+        license: licenses.find(license => license.license === values.license),
+        creators: values.creators,
+        processors: values.processors,
+        rightsholders: values.rightsholders,
+      },
       agreementId: values.agreementId,
       metaImageAlt: values.metaImageAlt,
       metaImageId: values.metaImageId,
+      metaImage: {
+        id: values.metaImageId,
+        alt: values.metaImageAlt,
+      },
```

Comment:
Skal det her være dobbelt opp med verdier?

Reply:
Det er en forenkling for å slippe å endre for mye. Strukturen som trengs ved innsending til apiet er annerledes enn det som brukes av formet. Jeg kunne laget en til funksjon og byttet mellom dem, men mengden endringer bare øker, og det må sikkert endres igjen om vi skal innføre ts her i fremtiden uanset

Reply:
Jeg mener prinsipielt at vi burde ordne det slik at de bruker samme struktur, men jeg kan se at det fort blir mye arbeid så la gå 👍

Reply:
Helt enig at dette burde ordnes, men føler det passer bedre i en ts oppgradering. Da blir det også lettere å se om vi har vert konsekvente.
Det var alleredet litt dobbelt opp her før, men jeg endret slik at de forskjellige datastrukturene hadde egne navn i stedet for å overskrive de samme feltene

Reply:
Jeg ser nå at kall mot concept-api sender med alle disse feltene, også de som kun brukes i formen.

Da vi diskuterte omskriving av conceptApi til typescript var planen at det skulle være egne interfaces og transform-funksjoner for APIet og for formen, da de ikke er helt kompatible. Er det bedre å reversere til strukturen som er i master nå, eller går det an å unngå at feltene i formen og APIet flyter over i hverandre?

Reply:
Skulle helst egentlig skrevet om hele formet til ts

Reply:
Problemet med hvordan det er i master er at ting transformeres mer enn nødvendig.
Jeg endret getConcept nå til å bare brukes til å transformere form konseptet til api konseptet. Burde ikke trenge noen flere transform funksjoner da. Transfomeringen den andre veien gjøres i getInitValues.

Om man vil gjøre transformeringen et annet sted er det greit, men master har også disse to funksjonene så :shrug: